### PR TITLE
Fix unit tests to run on Windows

### PR DESCRIPTION
**Pull request by RodEsp**
### What does this PR do?
* Fixes a JSON parsing function to run correctly on Windows
* Adds a vscode launch configuration for debugging unit tests

### What issues does this PR fix or reference?
@W-8827814@

### CODE REVIEW

**Review by mshanemc — APPROVED** [review]

**Review by peternhale — APPROVED** [review]

File: .circleci/config.yml [hunk 1]
```diff
@@ -13,7 +13,11 @@ workflows:
           name: node-latest
           node_version: latest
       - release-management/test-package:
-          name: node-12
+          name: node-lts-active
+          node_version: '14'
+      - release-management/test-package:
+          name: node-lts-maintenance
+          node_version: '12'
```
Comment (RodEsp) [inline .circleci/config.yml:20]: What do you think of adopting something [like this](https://github.com/salesforcecli/plugin-auth/pull/80/files#diff-78a8a19706dbd2a4425dd72bdab0502ed7a2cef16365ab7030a5a0588927bf47R12-R20) for unit testing going forward?

Comment (peternhale) [reply .circleci/config.yml:20]: This requires a change each time new versions of node are created. Is there a simple way to supply symbolic names that are already well known or can be set from a single location.

**Review by peternhale — CHANGES_REQUESTED** [review]

File: .circleci/config.yml (hunk 1 above)
Comment (amphro) [reply .circleci/config.yml:20]: You would have to have another condition "if version == 'lts'" then add `--lts` to the nvm install command. See nvm's readme.

**Review by shetzel — APPROVED** [review]

**Comment by RodEsp** [comment]
Closing in favor of just merging https://github.com/salesforcecli/leif/pull/21 and letting leif take care of it.
Will continue to use this branch just for testing.

**Pull request closed without merging**